src/main/app/src/hooks/useKoodi/index.ts

```diff
@@ -13,7 +13,7 @@ const useKoodi = koodiUri => {
   // Not passing koodisto version parsed from koodiUri here, because we
   // want to use the latest koodisto. Also if a single koodi tries to fetch
   // koodisto again for an older koodisto versio this will return empty data.
```

Comment:
Jäiks tääkin kommentti turhaksi?

Reply:
Tämä kommentti on minusta edelleen tarpeellinen. Koodiston versiota ei edelleenkään välitetä ja on hyvä dokumentoida miksi niin ei tehdä.

Reply:
Eikun äh niin, se aiemmi oli defaulttina tyhjä ja nyt sitä ei vaan anneta eteenpäin :+1: